mautic/mautic
# Remove backend browser notifications

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      |  Y
| New feature?  | N
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  N/A

## Description

This PR removes the backend "Allow Push Notifications" ask that we do not use in Mautic.

## Steps to test this PR

Ensure that your Browser Notifications for your mautic install where you're going to test this is set to "Alway Ask" (click the lock/icon in the navbar, then select notifications to change it, or in the browser notification config in Chrome), refresh the page and it should ask you for permission again. Say yes, then change it back to to always ask again (so we can be sure it's working correctly). Then apply the PR and refresh the mautic backend. It should not ask you any more.
